fission-web-server/library/Fission/Web/Server/PowerDNS/Types.hs

```diff
@@ -0,0 +1,27 @@
+module Fission.Web.Server.PowerDNS.Types
+  ( URL (..)
+  , ApiKey (..)
+  ) where
+
+import           Fission.Prelude
+
+newtype URL = URL { getUrl :: Text }
```

Comment:
Yaaaas `newtype`!

Reply:
This instead of our existing `URL` type because it's less structured I guess?

Reply:
Yeah, we don't ever parse or manipulate this one...